packages/flutter/lib/src/painting/borders.dart

```diff
@@ -516,7 +516,7 @@ abstract class ShapeBorder {
   /// and `b` after `t=0.5`.
   ///
   /// {@macro dart.ui.shadow.lerp}
-  static ShapeBorder? lerp(ShapeBorder? a, ShapeBorder? b, double t) {
+static ShapeBorder? lerp(ShapeBorder? a, ShapeBorder? b, double t) {
```

Comment:
Please add back in the indentation so it's consistent with the rest of the file.